Source/GSBindingHelpers.h

```diff
@@ -78,4 +78,10 @@
 @interface GSKeyValueAndBinding : GSKeyValueBinding 
 @end
 
+@interface GSObservableArray : NSArray
```

Comment:
No, as I wrote above this is about KVO with key paths that include an array. At the moment mostly something like "arrangedObjects.parentTask.name" where "arrangedObjects" is a property of NSArrayController. As this is an NSArray it would normally not be observable. So we need a way around which gets provided by this class.
This is a rather clever concept but marginal for this pull request and also not fully working in GNUstep at the moment.